app/main/views/direct_award.py

```diff
@@ -0,0 +1,70 @@
+from datetime import datetime
+from dmutils import csv_generator
+from flask import abort, render_template, request, Response, url_for
+from flask_login import flash, current_user
+from .. import main
+from ..auth import role_required
+from ... import data_api_client
+
+@main.route('/direct-award/outcomes', methods=['GET'])
+@role_required('admin-ccs-category', 'admin-framework-manager', 'admin-ccs-sourcing')
+def download_outcomes():
```

Comment:
Can we have some test coverage for this view? There's similar tests for supplier and user CSV downloads that you can copy from.

Reply:
@katstevens - I've added tests for this now